test/FileCipher/CompatibilityTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace ZendTest\Crypt\FileCipher;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Zend\Crypt\FileCipher;
+use Zend\Crypt\Symmetric\Mcrypt;
+use Zend\Crypt\Symmetric\Openssl;
+use Zend\Math\Rand;
+
+class CompatibilityTest extends TestCase
+{
+    public function setUp()
+    {
+        if (! extension_loaded('mcrypt') || ! extension_loaded('mcrypt')) {
+            $this->markTestSkipped(
+                sprintf("I cannot execute %s without Mcrypt and OpenSSL installed", __CLASS__)
```

Comment:
@weierophinney why? We are not requiring ext-mcrypt or ext-openssl. We only suggest ext-openssl. There are many functions of Zend\Crypt that does not require these extensions.